.env.example

```
@@ -6,6 +6,8 @@ QISKIT_IBM_TOKEN= #QISKIT_IBM_TOKEN
 QISKIT_IBM_INSTANCE= #QISKIT_IBM_INSTANCE
 IONQ_API_KEY= #IONQ_API_KEY
 OQC_AUTH_TOKEN= #OQC_AUTH_TOKEN
+IQM_SERVER_URL= #IQM_SERVER_URL
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)